node-backend/app/routes/routes.js

```diff
@@ -6,11 +6,21 @@ const loginController = require("../controllers/loginController");
 require("../google_auth/google_auth")();
 const router = express.Router();
 router.use(bodyParser.urlencoded({ extended: true }));
-router
-  .route("/login")
-  .post(
-    passport.authenticate("google-token", { session: false }),
-    loginController.login
-  );
+router.post(
+  "/oauth/google",
+  (req, res, next) => {
+    if (req.body.access_token) {
+      next();
+    } else {
+      res.status(404).send({
```

Comment:
this should be 400 bad request and error message should be "missing/invalid access_token"